### PR TITLE
[new] create a fork using aiven cli

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Provides the instructions to create a fork using cli

validated as well:
```
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % avn service get grafana-45ce200
SERVICE_NAME     SERVICE_TYPE  STATE    CLOUD_NAME               PLAN         CREATE_TIME           UPDATE_TIME           NOTIFICATIONS
===============  ============  =======  =======================  ===========  ====================  ====================  =============
grafana-45ce200  grafana       RUNNING  timescale-aws-us-east-1  dashboard-1  2022-08-18T11:01:07Z  2022-08-24T07:15:06Z
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % avn service create grafana-test1 --project internal-90d0 -t grafana --plan dashboard-1 --cloud timescale-aws-us-east-1  -c service_to_fork_from=grafana-45ce200                                      
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % avn service list                                                                                                 
SERVICE_NAME                             SERVICE_TYPE  STATE       CLOUD_NAME                    PLAN                                             CREATE_TIME           UPDATE_TIME           NOTIFICATIONS
=======================================  ============  ==========  ============================  ===============================================  ====================  ====================  =============
dg-grafana-tutorial-test                 grafana       RUNNING     timescale-aws-eu-west-1       dashboard-1                                      2022-04-19T17:37:44Z  2022-08-24T06:39:45Z
grafana-2042745a                         grafana       POWEROFF    timescale-aws-us-east-1       dashboard-1                                      2022-04-18T14:00:36Z  2022-05-03T14:18:44Z
grafana-2bf10fd6                         grafana       POWEROFF    timescale-aws-us-east-1       dashboard-1                                      2022-03-24T18:04:25Z  2022-07-05T23:49:03Z
grafana-381252ae                         grafana       RUNNING     timescale-aws-us-east-1       dashboard-1                                      2022-08-22T13:02:17Z  2022-08-24T07:09:32Z
grafana-45ce200                          grafana       RUNNING     timescale-aws-us-east-1       dashboard-1                                      2022-08-18T11:01:07Z  2022-08-24T07:15:06Z
grafana-test1                            grafana       REBUILDING  timescale-aws-us-east-1       dashboard-1                                      2022-08-24T07:30:58Z  2022-08-24T07:30:58Z
```
# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Note: This PR uses a link to [aiven-client-install] that is a part of the [PR](https://github.com/timescale/docs/pull/1502/).
The link should resolve after the related PR is merged.

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: mst/aiven-client/create-fork.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+product: mst
+---
+
+# Create a fork of your service using Aiven Client
+
+When you a fork a service, you create an exact copy of the service, including
+the underlying database. You can use it to create a development copy of your
+production environment, set up a snapshot to analyze an issue or test an
+upgrade, or create an instance in a different cloud, geographical location, or
+under a different plan.
```
Comment (charislam) [inline mst/aiven-client/create-fork.md:13]: Non-blocking suggestion: set this up as bullets, easier to skim

File: mst/aiven-client/create-fork.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+You can switch to `project-fork` and view the newly created `grafana-fork` using:
+
+```bash
+   avn service list
+```
+
+[about-mst]: /mst/:currentVersion:/about-mst/
+[aiven-client-install]: /mst/aiven-client/aiven-client-install/
```
Comment (charislam) [inline mst/aiven-client/create-fork.md:73]:
```suggestion
[aiven-client-install]: /mst/:currentVersion:/aiven-client/aiven-client-install/
```

File: mst/aiven-client/create-fork.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+1.  Get the details of the service that you want to fork:
+
+    ```bash
+    avn service get <SERVICE_NAME>
+    ```
+
+1.  Create a fork of the service
+
+    ```bash
+    avn service create <NAME_OF_FORK> --project <PROJECT> -t <SERVICE_TYPE> --plan <PLAN> --cloud <CLOUD_NAME> -c service_to_fork_from=<NAME_OF_SERVICE_TO_FORK>
```
Comment (charislam) [inline mst/aiven-client/create-fork.md:49]: Non-blocking suggestion: split this onto multiple lines with line continuation (`\`) because otherwise it's quite a bit of scrolling to read the whole thing

File: mst/aiven-client/create-fork.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+    For more information about projects, plans, and other details about
+    services,see [Services][about-mst].
+
+</procedure>
+
+## Example
+
+To create a fork named `grafana-fork` for a service named `grafana` in `fork-project`
+on `timescale-aws-us-east-1` with the plan `dashboard-1`:
```
Comment (charislam) [inline mst/aiven-client/create-fork.md:60]: Non-blocking suggestion: This is hard to parse because prepositional phrase galore! Maybe try splitting into multiple sentences?

File: mst/aiven-client/create-fork.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    services,see [Services][about-mst].
+
+</procedure>
+
+## Example
+
+To create a fork named `grafana-fork` for a service named `grafana` in `fork-project`
+on `timescale-aws-us-east-1` with the plan `dashboard-1`: 
+
+```bash
```
Comment (charislam) [inline mst/aiven-client/create-fork.md:62]: Same comment about splitting onto multiple lines

**Review by charislam — CHANGES_REQUESTED** [review]
Needs a page-index update

File: mst/aiven-client/create-fork.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
+---
+title: Create a fork using Aiven Client 
+excerpt: Create a fork of the service using Aiven Command Line tool for fully managed services on AWS, Azure, or GCP.
+product: mst
+---
+
+# Create a fork of your service using Aiven Client
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:7]:
```suggestion
# Fork services with Aiven Client
```

File: mst/aiven-client/create-fork.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+When you a fork a service, you create an exact copy of the service, including
+the underlying database. You can use a fork of your service to:
+*   create a development copy of your production environment
+*   set up a snapshot to analyze an issue or test an upgrade
+*   create an instance in a different cloud, geographical location, or under
+    a different plan.
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:14]:
```suggestion

*   Create a development copy of your production environment.
*   Set up a snapshot to analyze an issue or test an upgrade.
*   Create an instance in a different cloud, geographical location, or under
    a different plan.
```

File: mst/aiven-client/create-fork.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+## Before you begin
+
+*   [Install Aiven Client and connect][aiven-client-install] to Managed Service
+    for TimescaleDB
+*   [Create a service][create-service] in Managed Service for TimescaleDB
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:20]:
```suggestion

Before you begin, make sure you have: 

*   Installed [Aiven Client][aiven-client-install].
*   Signed in to your [Managed Service for TimescaleDB dashboard][mst-login].
*   Created a [service][create-service] in Managed Service for TimescaleDB.
```

File: mst/aiven-client/create-fork.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+
+You can switch to `project-fork` and view the newly created `grafana-fork` using:
+
+```bash
+   avn service list
+```
+
+[about-mst]: /mst/:currentVersion:/about-mst/
+[aiven-client-install]: /mst/:currentVersion:/aiven-client/aiven-client-install/
+[create-service]: /install/:currentVersion:/installation-mst/#create-your-first-service
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:79]:
```suggestion
[create-service]: /install/:currentVersion:/installation-mst/#create-your-first-service
[mst-login]: https://portal.managed.timescale.com
```

File: mst/aiven-client/create-fork.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+
+*   [Install Aiven Client and connect][aiven-client-install] to Managed Service
+    for TimescaleDB
+*   [Create a service][create-service] in Managed Service for TimescaleDB
+
+<procedure>
+
+## Creating a fork of your service
+
+1.  Switch to the `Project` that contains the service that you want to fork:
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:26]:
```suggestion
1.  In the Aiven Client, log in to your Managed Service for TimescaleDB account.

1.  Switch to the project that contains the service you want to fork:
```

File: mst/aiven-client/create-fork.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+
+    ```bash
+     avn switch <PROJECT>
+    ```
+
+1.  List the services in the `Project`:
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:32]:
```suggestion
1.  List the services in the project:
```

File: mst/aiven-client/create-fork.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+
+    ```bash
+     avn service list
+    ```
+
+1.  Make a note of the service that you want to fork, listed under
+    `SERVICE_NAME` column in the output.
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:39]: This is working with the output of the previous step, so these could be combined.

File: mst/aiven-client/create-fork.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+1.  Get the details of the service that you want to fork:
+
+    ```bash
+    avn service get <SERVICE_NAME>
+    ```
+
+1.  Create a fork of the service
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:47]:
```suggestion
1.  Create the fork:
```

File: mst/aiven-client/create-fork.md [hunk 14]
```diff
@@ -0,0 +1,79 @@
...
+
+    ```bash
+    avn service create <NAME_OF_FORK> --project <PROJECT>\
+    -t <SERVICE_TYPE> --plan <PLAN> --cloud <CLOUD_NAME>\
+    -c service_to_fork_from=<NAME_OF_SERVICE_TO_FORK>
+    ```
+
+    For more information about projects, plans, and other details about
+    services,see [Services][about-mst].
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:56]: This feels a bit generic, I'm not sure it's required here. Perhaps if you feel it's important, it could go in the concept info at the top of the page instead?

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: mst/aiven-client/create-fork.md [hunk 15]
```diff
@@ -8,28 +8,33 @@ product: mst
...
+*   Installed [Aiven Client][aiven-client-install].
+*   Signed in to your [Managed Service for TimescaleDB dashboard][mst-login].
+*   Created a [service][create-service] in Managed Service for TimescaleDB.
 
 <procedure>
 
 ## Creating a fork of your service
 
-1.  Switch to the `Project` that contains the service that you want to fork:
+1.  In the Aiven Client, log in to your Managed Service for TimescaleDB account.
```
Comment (Loquacity) [inline mst/aiven-client/create-fork.md:29]:
```suggestion
1.  In the Aiven Client, log in to your [Managed Service for TimescaleDB dashboard][mst-login].
```

**Review by Loquacity — COMMENTED** [review]
Just one little thing, otherwise LGTM

**Review by Loquacity — APPROVED** [review]

**Review by charislam — APPROVED** [review]

**Comment by alexthinath** [comment]
I think we need an actual engineer to test this one. We have issues with the CLI so I'd prefer we double check it

**Comment by Loquacity** [comment]
@Rajakavitha1 I think we can go ahead and merge this one.

**Review by Loquacity — APPROVED** [review]
